src/graphics.cpp

```diff
@@ -495,3 +217,7 @@ int Graphics::GetDefaultFps() {
 MessageOverlay& Graphics::GetMessageOverlay() {
 	return *message_overlay;
 }
+
+Transition *Graphics::GetTransition() {
```

Comment:
Is it possible that Transition is nullptr? If not you can do it like MessageOverlay above and return by & reference.